gulpfile.js

```diff
@@ -111,16 +111,18 @@ const copy = {
       colors.blue,
       `Copy USWDS fonts: ${getSrcFrom("fonts")} → ${paths.dist.fonts}`
     );
-    return src(`${getSrcFrom("fonts")}/**/**`.replace("//", "/")).pipe(
-      dest(paths.dist.fonts)
-    );
+    return src(`${getSrcFrom("fonts")}/**/**`.replace("//", "/"), {
+      encoding: false,
+    }).pipe(dest(paths.dist.fonts));
```

Comment:
Avoids corruption on images and fonts.

More info here: https://github.com/gulpjs/gulp/issues/2803